trino/Dockerfile

```diff
@@ -78,5 +84,53 @@ COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/trino-server-${PRODUCT}
 # ===
 
+FROM stackable/image/java-devel AS jmx-exporter-builder
+
+ARG JMX_EXPORTER
+ARG JMX_EXPORTER_GIT_REV
+
+WORKDIR /stackable
+
+RUN git clone https://github.com/prometheus/jmx_exporter --branch=${JMX_EXPORTER} --depth=1 && \
+    cd jmx_exporter && git checkout ${JMX_EXPORTER_GIT_REV}
+RUN --mount=type=cache,target=/root/.m2/repository cd jmx_exporter && \
+    mvn package
+
+FROM stackable/image/java-base
```

Comment:
```suggestion
# Not tagging base image because it is built as part of the same process
# hadolint ignore=DL3006
FROM stackable/image/java-base
```